src/mas/devops/templates/pipelinerun-install.yml.j2

```diff
@@ -713,3 +713,96 @@ spec:
     - name: shared-certificates
       secret:
         secretName: pipeline-certificates
+
+{%- if install_aibroker is defined and install_aibroker != "" %}
+
+    # IBM Maximo Location Services for Esri (arcgis)
+    # -------------------------------------------------------------------------
+    - name: install_aibroker
+      value: "{{ install_aibroker }}"
+    - name: mas_aibroker_channel
+      value: "{{ mas_aibroker_channel }}"
+{%- endif %}
```

Comment:
To ensure  the template renders cleanly note how the existing sections follow this pattern:

```
{%- endif %}
{%- if x %}
<empty line>
<conditional content header>
<conditional content>
{%- endif %}
{%- if y %}
<empty line>
<conditional content header>
<conditional content>
{%- endif %}
```

By including the empoty line preceeding the header in the conditional block it avoids creating yaml that if filled with masses of empty lines when the conditionals are not met.

You should be able to run mas install using your build of this package and verify that basically there's no change in the generated pipelinerun compared to before when none of the aibroken settings are provided.

Reply:
fixed pattern